packages/runtime/agent-scheduler/src/scheduler.ts

```diff
@@ -217,6 +217,9 @@ class AgentScheduler extends TypedEventEmitter<IAgentSchedulerEvents> implements
         // May be we want a randomized timer (Something like raft) to reduce chattiness?
         // eslint-disable-next-line @typescript-eslint/no-misused-promises
         this.consensusRegisterCollection.on("atomicChanged", async (key: string, currentClient: string | null) => {
+            // emit on current client change
+            this.emit((!currentClient) ? "taskPicked" : "taskLost", key);
```

Comment:
There is already "picked" & "released" events, this seems to duplicate them. Why?

Reply:
My bad, I didn't realise that it's already there

Reply:
Hi @vladsud,

I check that again but  "picked" & "released" not emitted to all connected client it just emit with condition currentClient === this.clientId

but we want this to be emitted for all connected client.